.travis.yml

```diff
@@ -15,7 +15,7 @@ env:
   - TEST_SUITE=unit
 before_install:
   - sudo apt-get -qq update
-  - sudo apt-get install -y python3-coverage python3-distutils-extra
+  - sudo apt-get install -y python3-coverage python3-distutils-extra squashfs-tools
```

Comment:
Or I don't understand why you need it here if we just use travis for unit tests.